example/advection/advection_package.cpp

```diff
@@ -381,14 +381,14 @@ template <typename T>
 Real AdvectionHst(MeshData<Real> *md) {
   auto pmb = md->GetBlockData(0)->GetBlockPointer();
 
+  const auto ib = pmb->cellbounds.GetBoundsI(IndexDomain::interior);
+  const auto jb = pmb->cellbounds.GetBoundsJ(IndexDomain::interior);
+  const auto kb = pmb->cellbounds.GetBoundsK(IndexDomain::interior);
```

Comment:
(Independent of this PR) this change in lines makes we wonder if `cellbounds` should not be at an even more central place (given that they are fixed "globally" for any given simulation).

Reply:
Actually after this PR I noticed it already exists: `MeshData` and `MeshBlockData` both provide `GetBounds*`